clouddriver-azure/src/main/groovy/com/netflix/spinnaker/clouddriver/azure/resources/servergroup/model/AzureServerGroupDescription.groovy

```diff
@@ -193,6 +194,15 @@ class AzureServerGroupDescription extends AzureResourceOpsDescription implements
     azureSG.region = scaleSet.location()
     azureSG.upgradePolicy = getPolicyFromMode(scaleSet.upgradePolicy().mode().name())
 
+    def termProfile = scaleSet.virtualMachineProfile()?.scheduledEventsProfile()?.terminateNotificationProfile()
+    if (termProfile)
+    {
+      String[] str = termProfile.notBeforeTimeout().findAll( /\d+/ )
+      if (str.size() > 0) {
+        azureSG.terminationNotBeforeTimeout = str[0].toInteger()
```

Comment:
Again since we are assuming that this in minutes we probably have to the conversion if other representations are possible. What happens in scenario where PT600S was provided during install, does the saved config give us back as PT10M ? then this is ok

Reply:
```
--terminate-notification-time    : Length of time (in minutes, between 5 and 15) a notification to be sent to the VM on the instance metadata server till the VM gets deleted.
```

So az cli only accepts minutes, i’m fine with forcing Spinnaker to also only accept minutes, i’ll update the PR to state minutes.  This also has a limit but i haven't seen this type of limit implemented anywhere else in clouddriver, if you have an opinion on it.